### DISCUSSION TITLE
submit.html produces 404

### DISCUSSION THREAD

**Discussion by info-sic** [pandora-analysis/pandora#447] [Q&A] [answered]
Hi all,
my homepage url https://a.b.c.d:6100/submit produces an 404 below the navbar
file submit.html exists in the templates directory.

I edited the submit.html to add our logo.  It did work for some time but eventualy it 404ed.
I have no clue, how to debug that?

There is an error in  pandora_error.log, which I am unable to c&p right now (vm)  with a reference to virtualenv in ~/.cache/pypoetry about an illegal \t character while loading tokens in scanner.py . Is that related?

TIA
Manu

**Comment by Rafiot** [comment]
This is very hard to figure out what the problem might be without seeing the code you changed. If you want more help, please share a diff of your changes.

You should put the logo in `website/web/static/images/`.

Note that `scanner.py` isn't a file in the project, so I have no idea.

**Comment by info-sic** [comment, marked as answer]
Ok, found it. If you learn to read, it helps...
I changed the blocklist.yml to include more M$-Office malware and accidently inserted a line with a single  TAB \t.
That caused the parsing in helpers.py to fail miserably, because /lib/python3.10/site-packages/yaml/scanner.py which does the actual parsing ,doesn't handle single tabs very well.
Well, the error is pretty clear: "found character '\t' that cannot start any token" it just took a long time, to find out in which file this \t was hidden.
Case closed. Thanx for your help!
Manu
